### ISSUE TITLE
Purging Specific cache

### ISSUE THREAD

**Issue by TestCandidate** [kixelated/steamcache#3]
I couldn't make sense of the cached files.

Is there a way to purge cache for a specific game or a specific server / gaming platform?

**Comment by kixelated** [comment]
Not possible at the moment because the cache is shared. You could purge individual URLs but that's not particularly useful.

Alternatively, you could change the code to make a separate cache per platform. proxy_cache_path will define a cache and some other directive will choose which cache to use (I forget).

**Comment by TestCandidate** [comment]
Yes i suppose dividing each platform to save in a seperate folder is a way to go. 

Just out of curiosity, how can u purge a specific uri? Maybe i can track some urls for a specific patch or game then i can write a script to purge anything similar
